R/obo.R

```diff
@@ -81,7 +81,7 @@ getDefaultCVVersion = function()
 #'   cv_dict = CV_$new() ## uses 'getCVDictionary()' to populate the singleton
 #'   cv_2 = CV_$new() ## uses the same data without parsing again
 #'
-#' Wherever you need this data, simply re-grab the singleton using 'CV_$new()$data'
+#' Wherever you need this data, simply re-grab the singleton using 'CV_$new()$data' (or use the convenience function getCVSingleton() from outside the package)
```

Comment:
Why is  it outside the package?

Reply:
The function allows you to get the MS CV which is shipped with rmzqc. This might be useful to avoid duplicate CVs being used in downstream packages, e.g. PTXQC